core/hosting/HostWithCoreClrHost/src/SampleHost.cpp

```diff
@@ -92,12 +92,12 @@ int main(int argc, char* argv[])
     // <Snippet2>
     coreclr_initialize_ptr initializeCoreClr = (coreclr_initialize_ptr)GetProcAddress(coreClr, "coreclr_initialize");
     coreclr_create_delegate_ptr createManagedDelegate = (coreclr_create_delegate_ptr)GetProcAddress(coreClr, "coreclr_create_delegate");
-    coreclr_shutdown_ptr shutdownCoreClr = (coreclr_shutdown_ptr)GetProcAddress(coreClr, "coreclr_shutdown");
```

Comment:
Is there a reason to use coreclr_shutdown_2 instead of coreclr_shutdown if we're not using the latchedExitCode parameter?

Reply:
Yeah you're right. It's OK to use `coreclr_shutdown`. I'll revert the changes.